mutation/mutation.cc

```diff
@@ -176,6 +176,16 @@ mutation mutation::compacted() const {
     return m;
 }
 
+size_t mutation::memory_usage(const ::schema& s) const {
+    auto res = sizeof(*this);
+    if (_ptr) {
+        res += sizeof(data);
+        res += _ptr->_dk.memory_usage();
+        res += _ptr->_p.memory_usage(s);
+    }
```

Comment:
doesn't this result in double accounting of size of decorated_key and mutation_partition which their memory_usage() already account for their own size? if so, I suggest we do res += sizeof(data::_schema) instead of the whole data. other than that, looks good to me.

Reply:
Yes, I should have written `external_memory_usage()`, which excludes `sizeof(*this)`.

Reply:
Fixed in v2.

Reply:
It still says _p->memory_usage()

Reply:
> It still says _p->memory_usage()

Yes, I found this awkward given that Botond updated the branch for addressing it. If you go to the branch, you'll see the change. If you also click on "compare" https://github.com/scylladb/scylladb/compare/c1e791f4c9bf7a5c502a7196049fe766b10fdbd5..2f44b08126fc8b93368fc48843309d85ee725dbc, you'll also see it.

Reply:
eventual consistency?